docusaurus/docs/drafts/_category_.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I question whether we want to keep this folder and its content generally, open issues for content seems sufficient

Reply:
Agreed.